### ISSUE TITLE
Question about validation

### ISSUE THREAD

**Issue by CeciliaPYY** [jfzhang95/pytorch-deeplab-xception#69]
Hello, first thanks for your pytorch version of deeplab v3+, I try it at last days of 2018, and I got a perfect model whose performance is great both on train and validation datasets, which means their loss are steadily decreasing at my first try. However, when I give it a second try, train loss of the new model still decreased but the validation loss went up and down sometimes. 

Below, is the first try performance.
train
![image](https://user-images.githubusercontent.com/30582437/54107723-ec89cf80-4414-11e9-950b-29fe939e5ef0.png)

validation
![image](https://user-images.githubusercontent.com/30582437/54107740-f7dcfb00-4414-11e9-9f10-fbfefd39c5ae.png)

Below, is the second try performance.
train
![image](https://user-images.githubusercontent.com/30582437/54107543-75ecd200-4414-11e9-8015-a4ac59043db9.png)

validation
![image](https://user-images.githubusercontent.com/30582437/54107564-843aee00-4414-11e9-8316-33b0f8f43646.png)

ps, the dataset of two try is definitely the same!!

**Comment by sonack** [comment]
did you used the syncd bn option？

**Comment by CeciliaPYY** [comment]
> did you used the syncd bn option？

I don't think I used it cause I just have one gpu to train it.

**Comment by Ellsionjeep** [comment]
@CeciliaPYY  How to get kinds of  figures ,can you give me a copy? thanks
